### DISCUSSION TITLE
Differing Isotype calls between MiXCR and Immcantation

### DISCUSSION THREAD

**Discussion by abspangler13** [milaboratory/mixcr#1867] [Q&A]
Hello,

I am processing some smart-seq b cell data using the smart-seq2-vdj preset.  After this, I create a fasta file using the targetSequences output by MiXCR and do another alignment using Immcanation's `AssignGenes.py` function which uses IgBlast to align to the IMGT database. I'm seeing a lot of differing isotype calls between the MiXCR and Immcantation outputs and I'm wondering what is causing this.  Could it be due solely to the different reference databases MiXCR and Immcantation use for the alignment? Below, I've shown an example of the data where the first two columns are the calls made by Immcantation and the last column is the call made by MiXCR. It's strange to me that so many sequences called IGHG2 by Immcantation would be called IGHG1 by MiXCR. 

<img width="469" alt="image" src="https://github.com/user-attachments/assets/999ca6e9-15f9-402f-b09b-4e7b010267db" />

**Comment by mizraelson** [comment]
In most cases, I see that the top hit (the one with the highest score) aligns with what you get from Immcanation. For others, the difference could indeed be due to the library they use. If you can share the sequence itself, I can align it and investigate why a certain isotype is being returned.
